go-controller/hack/lint.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
+set -x
 
-GO111MODULE=on golangci-lint run \
+GOPATH=${GOPATH:-$(go env GOPATH)}
+curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b ${GOPATH}/bin
```

Comment:
I'd really rather not execute this every time I lint...

Reply:
now checks for that file before downloading it again. But didn't your original PR do the same thing too, or is Make smart enough to not re-curl the file somehow?

Reply:
@squeed do you mean the curl or the `go env GOPATH`?